packages/ocapn/test/codecs/passable.test.js

```diff
@@ -1,83 +1,167 @@
 // @ts-check
 
 /**
- * @import { SyrupCodec } from '../../src/syrup/codec.js'
- * @import { SyrupReader } from '../../src/syrup/decode.js'
- * @import { SyrupWriter } from '../../src/syrup/encode.js'
  * @import { CodecTestEntry } from './_codecs_util.js'
- * @import { Settler } from '@endo/eventual-send'
  */
 
 import test from '@endo/ses-ava/test.js';
 
 import { makeTagged } from '@endo/pass-style';
 import { makeSyrupReader } from '../../src/syrup/decode.js';
+import { makeSyrupWriter } from '../../src/syrup/encode.js';
 import { makeSelector } from '../../src/selector.js';
-import { sel, str, bts, bool, int, list, btsStr } from './_syrup_util.js';
-import { makeCodecTestKit, testBidirectionally } from './_codecs_util.js';
+import { recordSyrup } from './_syrup_util.js';
+import {
+  exporterLocation,
+  makeCodecTestKit,
+  runTableTests,
+} from './_codecs_util.js';
 import { throws } from '../_util.js';
-import { uint8ArrayToImmutableArrayBuffer } from '../../src/buffer-utils.js';
+import {
+  immutableArrayBufferToUint8Array,
+  uint8ArrayToImmutableArrayBuffer,
+} from '../../src/buffer-utils.js';
+import { encodeSwissnum } from '../../src/client/util.js';
+import { getSturdyRefDetails } from '../../src/client/sturdyrefs.js';
 
 const textEncoder = new TextEncoder();
 
 const { PassableCodec } = makeCodecTestKit();
 
+// The PassableCodec covers References (Targets and Promises), but these tests do not,
+// as it would entrain the entire client (especially for third-party references). They are tested
+// in the client tests.
+
 /** @type {CodecTestEntry[]} */
 const table = [
-  { syrup: `<${sel('void')}>`, value: undefined },
-  { syrup: `<${sel('null')}>`, value: null },
-  { syrup: `${bool(true)}`, value: true },
-  { syrup: `${bool(false)}`, value: false },
-  { syrup: `${int(123)}`, value: 123n },
-  { syrup: `${str('hello')}`, value: 'hello' },
-  {
-    syrup: btsStr('hello'),
+  { name: 'undefined', value: undefined },
+  { name: 'null', value: null },
+  { name: 'boolean true', value: true },
+  { name: 'boolean false', value: false },
+  { name: 'integer 123', value: 123n },
+  { name: 'string hello', value: 'hello' },
+  {
+    name: 'byte array hello',
     value: uint8ArrayToImmutableArrayBuffer(
       new Uint8Array([0x68, 0x65, 0x6c, 0x6c, 0x6f]),
     ),
   },
   {
-    syrup: bts(
-      uint8ArrayToImmutableArrayBuffer(
-        new Uint8Array([0x68, 0x65, 0x6c, 0x6c, 0x6f]),
-      ),
-    ),
+    name: 'byte array',
     value: uint8ArrayToImmutableArrayBuffer(
       new Uint8Array([0x68, 0x65, 0x6c, 0x6c, 0x6f]),
     ),
   },
   {
-    syrup: `${sel('hello')}`,
+    name: 'selector',
```

Comment:
Will need a note/issue to track follow-up to go back to Symbol, since we got consensus on that. Not in scope for your change.